mgr/src/cmds/nodes.cr

```diff
@@ -9,7 +9,7 @@ class Args
 end
 
 command "node.add", "Add a node to Kadalu Storage pool" do |parser, args|
-  parser.banner = "Usage: kadalu node add POOL/NAME ENDPOINT [arguments]"
+  parser.banner = "Usage: kadalu node add POOL/NAME [ENDPOINT] [arguments]"
```

Comment:
You can remove this `[ENDPOINT]` since it will be listed as argument (See next line)

Reply:
Ack! Done.